lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -156,7 +156,7 @@ public static function convertPositionalToNamedPlaceholders($statement)
 
         if ($currentLiteralDelimiter) {
             throw new OCI8Exception(sprintf(
-                'The statement contains non-terminated string literal starting at offset %d',
+                'The statement contains non-terminated string literal starting at offset "%d".',
```

Comment:
I don't think numbers need to be enclosed in quotes. Syntactically, quotes indicate a string which is not the case.